.doc_gen/cross-content/cross_LambdaForBrowser_JavaScriptV3_block.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "file://zonbook/docbookx.dtd"[
+        <!ENTITY % phrases-shared SYSTEM "file://AWSShared/common/phrases-shared.ent">
+        %phrases-shared;
+        ]>
+<block>
+    <para>
+        You can create a browser-based application that uses an &LAMlong; function to update a &DDBlong;
```

Comment:
**an** &DDBlong;
(long version expands to Amazon DynamoDB)

Reply:
done